### DISCUSSION TITLE
No images if the website is accessed without www / How to change the Favicon

### DISCUSSION THREAD

**Discussion by HenryRitalin** [LycheeOrg/Lychee#2544] [Q&A] [answered]
Hey Guys,
i have two little problems with Lychee 5.3.0.

The first one:
If I go to the site via https://testtest.com, everything works fine.
But if I go out via https://www.testtest.com, the images are not displayed and I get the error under Diagonese 
"Error: APP_URL (https://te*************st.com) does not match the current url (https://www.te***********st.com).
          This will break WebAuthn authentication.
    Error: APP_URL (https://www.te*************st.com) does not match the current url (https://www.te***********st.com).
          This will prevent images from being properly displayed."

I manage to change the behavior by editing the .env, then the page with www is displayed correctly, but the one without not.
Does anyone have an idea or even a solution to my problem?

The second problem:
When I change the FAVICON, it does not appear in the browser. I changed it in the /public/ folder, but to no avail.

Thanks Guys.

**Comment by d7415** [comment, marked as answer]
> Does anyone have an idea or even a solution to my problem?

Pick one. Redirect the other one to the correct one.



> When I change the FAVICON, it does not appear in the browser. I changed it in the /public/ folder, but to no avail.

Have you cleared your browser cache?

**Reply by HenryRitalin** [reply]
My hoster doesn't differentiate between www and without. So I can't set up a forwarding. Annoying.
changes in the HTACCESS dont work

The FAVICON thing solved itself, no idea how - because even when I cleared the browser cache it still didn't show up. Now it works
